bower.json

```diff
@@ -19,7 +19,7 @@
   ],
   "main": "./dist/angular-bootstrap-switch.js",
   "dependencies": {
-    "angular": "~1.4.0",
```

Comment:
@icholy I don't think so.
By `~1.5` I'd have only enabled support for **1.5** and not for **1.4**.
By using `~1.4` I enable support for both the newest version (so **1.5** atm.) and also the older one (**1.4**).

Nice explaination [here](http://stackoverflow.com/a/20412378)